config/checkstyle/checkstyle.xml

```diff
@@ -24,7 +24,9 @@
         <module name="CatchParameterName">
             <property name="format" value="^e$"/>
         </module>
-        <module name="ClassDataAbstractionCoupling"/>
+        <module name="ClassDataAbstractionCoupling">
+            <property name="max" value="8"/>
```

Comment:
That's one way to do it, sure. But then you should also review where this check has been disabled previously and whether these suppressions can be removed.

Alternatively, disable the check where it's failing now. We're going to review the whole Checkstyle configuration at some point.

Reply:
Good advise - wasn't aware of that possibility. Disabled the check.